### ISSUE TITLE
Would it please be possible to tag/release new versions?

### ISSUE THREAD

**Issue by sanjayankur31** [BlueBrain/mod2c#46]
Hello,

The latest tag here on Github is 0.9. The latest version, from the CMakeLists.txt file, however is 2.1.0. Would it please be possible to tag/release the new versions? It just makes it easier for us to package nmodl downstream at [NeuroFedora](https://neuro.fedoraproject.org).

Cheers,

**Comment by alkino** [comment]
I try to push tag following CMakeLists.txt but dates are weird: https://github.com/BlueBrain/mod2c/tags

**Comment by sanjayankur31** [comment]
Thanks, the dates indeed are odd---perhaps the git history was a bit messed up?

**Comment by pramodk** [comment]
@sanjayankur31  : we stopped tagging releases on this repo because this is now used as submodule inside CoreNEURON (and not supposed to be installed independently). 

We are moving to new code generation framework https://github.com/BlueBrain/nmodl and hence very little changes to this repository.

**Comment by sanjayankur31** [comment]
@pramodk : yes, I noticed that. The issue with the submodule system, at least with Github, is that they're not included in the Github generated release/tagged tars (or weren't the last time I'd checked). This means that all downstream users have to manually download the tars from the different sub-module projects and then extract them in the right places, and it just makes it easier if tagged versions are used in CoreNeuron so that users can go fetch them without having to figure out what commit of the sub-module is used and fetch that from GitHub etc.

Is nmodl also similar then, and only to be used with CoreNeuron, and not as a standalone tool/library?

**Comment by alkino** [comment]
I think that we always build after a clone (maybe shallow), and never from a tarball. That's why we didn't see this problem.

**Comment by pramodk** [comment]
@alkino  is right! 

> Is nmodl also similar then, and only to be used with CoreNeuron, and not as a standalone tool/library?

NMODL is generic purpose tool and used outside CoreNEURON. So definitely we will releases regularly (new one soon).
